lib/Base/IO.cpp

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "glow/Base/IO.h"
+
+#include <cstdio>
+#include <sys/stat.h>
+#include <sys/types.h>
+
+namespace {
+
+static size_t filesize(std::string filename) {
+  struct stat st;
+  if (stat(filename.c_str(), &st) != 0) {
+    GLOW_ASSERT(false && "stat failed");
+  }
+  return (size_t)st.st_size;
+}
+
+} // namespace
+
+namespace glow {
+
+void writeToFile(const Tensor &T, std::string filename) {
+  FILE *fp = fopen(filename.c_str(), "wb");
+  GLOW_ASSERT(fp);
+  fwrite(T.getUnsafePtr(), T.getType().getElementSize(), T.size(), fp);
```

Comment:
just a random thought, maybe return success/failure from this method depending on how many bytes written (whether it match the expected number of bytes written).